keyboards/brick65/keymaps/default/keymap.c

```diff
@@ -0,0 +1,49 @@
+        KC_CAPS,    KC_A,     KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,            KC_ENT,         
+        KC_LSFT,              KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT,             KC_UP, 
+        KC_LCTL,    KC_LWIN,  KC_LALT,                   KC_SPC,           KC_SPC,           KC_SPC,  KC_RALT, KC_RCTL,            KC_LEFT,   KC_DOWN,   KC_RGHT
+    ),
+    [1] = LAYOUT(
+        KC_TILD,    KC_F1,    KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,   KC_F12,   KC_TRNS,   KC_TRNS,   KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,             KC_TRNS,   KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS,         
+        KC_TRNS,              RGB_TOG, RGB_MOD, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE,  KC_TRNS,             KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS,                   KC_TRNS,          KC_TRNS,          KC_TRNS, RGB_MOD, KC_TRNS,            KC_TRNS,   KC_TRNS,   KC_TRNS
+    ),
+    [2] = LAYOUT(
+        KC_TRNS,    KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,             KC_TRNS,   KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS,         
+        KC_TRNS,              KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,             KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS,                   KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS,   KC_TRNS,   KC_TRNS
+    ),
+    [3] = LAYOUT(
+        KC_TRNS,    KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,             KC_TRNS,   KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS,         
+        KC_TRNS,              KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,             KC_TRNS, 
+        KC_TRNS,    KC_TRNS,  KC_TRNS,                   KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS,   KC_TRNS,   KC_TRNS
+    )
```

Comment:
Empty layers are not needed, and needlessly increase firmware size.  They should be removed.
```suggestion
```